core/src/main/java/com/linecorp/armeria/client/limit/ConcurrencyLimitingExceedException.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.client.limit;
+
+/**
+ * A {@link RuntimeException} raised when {@link ConcurrencyLimitingClient#numActiveRequests()} exceeds
+ * the configured {@code maxConcurrency}.
```

Comment:
It's impossible to exceed the max concurrency with `ConcurrencyLimitingClient`. The exception is raised only when the client was not able to send a request until it reaches at the timeout. Could you rename this class and rephrase the Javadoc?

Reply:
The `ConcurrencyLimitingExceedException` is changed into `PendingRequestTimeoutException`. What do you think about this name? 🧐

Reply:
I like the new name. :+1: